### DISCUSSION TITLE
Installation issues while installing wget

### DISCUSSION THREAD

**Discussion by Bashert1601** [Homebrew/discussions#1160] [Getting started]
brew install wget
==> Searching for similarly named formulae...
Error: No similarly named formulae found.
Error: No available formula or cask with the name "wget".
==> Searching for a previously deleted formula (in the last month)...
Error: No previously deleted formula found.
==> Searching taps on GitHub...
Error: No formulae found in taps.

**Comment by SMillerDev** [comment]
What do `brew config` and `brew doctor` say?

**Reply by Bashert1601** [reply]
brew config
HOMEBREW_VERSION: 3.0.10-42-ge5bda4e
ORIGIN: https://github.com/Homebrew/homebrew-core
HEAD: e5bda4e781bdd74240b2f01df0f01ab5477e3300
Last commit: 8 hours ago
Core tap ORIGIN: https://github.com/Homebrew/homebrew-core
Core tap HEAD: e5bda4e781bdd74240b2f01df0f01ab5477e3300
Core tap last commit: 8 hours ago
Core tap branch: master
HOMEBREW_PREFIX: /usr/local
HOMEBREW_CASK_OPTS: []
HOMEBREW_MAKE_JOBS: 4
Homebrew Ruby: 2.6.3 => /usr/local/Homebrew/Library/Homebrew/vendor/portable-ruby/2.6.3_2/bin/ruby
CPU: quad-core 64-bit broadwell
Clang: 12.0 build 1200
Git: 2.24.3 => /Library/Developer/CommandLineTools/usr/bin/git
Curl: 7.64.1 => /usr/bin/curl
macOS: 10.15.6-x86_64
CLT: 12.4.0.0.1.1610135815
Xcode: N/A

brew doctor
Please note that these warnings are just used to help the Homebrew maintainers
with debugging if you file an issue. If everything you use Homebrew for is
working fine: please don't worry or file an issue; just ignore this. Thanks!

Warning: Suspicious https://github.com/Homebrew/brew git origin remote found.
The current git origin is:
  https://github.com/Homebrew/homebrew-core

With a non-standard origin, Homebrew won't update properly.
You can solve this by setting the origin remote:
  git -C "/usr/local/Homebrew" remote set-url origin https://github.com/Homebrew/brew

Warning: "config" scripts exist outside your system or Homebrew directories.
`./configure` scripts often look for *-config scripts to determine if
software packages are installed, and which additional flags to use when
compiling and linking.

Having additional scripts in your path can confuse software installed via
Homebrew if the config script overrides a system or Homebrew-provided
script of the same name. We found the following "config" scripts:
  /Library/Frameworks/Python.framework/Versions/3.9/bin/python3.9-config
  /Library/Frameworks/Python.framework/Versions/3.9/bin/python3-config

**Reply by SMillerDev** [reply]
> You can solve this by setting the origin remote:
git -C "/usr/local/Homebrew" remote set-url origin https://github.com/Homebrew/brew

Brew doctor has your fix.

**Reply by Bashert1601** [reply]
Here is what i got after running that command

brew install wget 
Updating Homebrew...
==> Homebrew is run entirely by unpaid volunteers. Please consider donating:
  https://github.com/Homebrew/brew#donations
==> Auto-updated Homebrew!
Updated 1 tap (homebrew/cask).
==> Updated Casks
Updated 1 cask.

==> Searching for similarly named formulae...
Error: No similarly named formulae found.
Error: No available formula or cask with the name "wget".
==> Searching for a previously deleted formula (in the last month)...
Error: No previously deleted formula found.
==> Searching taps on GitHub...
Error: No formulae found in taps.

**Reply by Bashert1601** [reply]
Now i get this:

brew doctor
Please note that these warnings are just used to help the Homebrew maintainers
with debugging if you file an issue. If everything you use Homebrew for is
working fine: please don't worry or file an issue; just ignore this. Thanks!

Warning: Suspicious https://github.com/Homebrew/homebrew-core git origin remote found.
The current git origin is:
  https://github.com/Homebrew/brew

With a non-standard origin, Homebrew won't update properly.
You can solve this by setting the origin remote:
  git -C "/usr/local/Homebrew/Library/Taps/homebrew/homebrew-core" remote set-url origin https://github.com/Homebrew/homebrew-core

Warning: "config" scripts exist outside your system or Homebrew directories.
`./configure` scripts often look for *-config scripts to determine if
software packages are installed, and which additional flags to use when
compiling and linking.

Having additional scripts in your path can confuse software installed via
Homebrew if the config script overrides a system or Homebrew-provided
script of the same name. We found the following "config" scripts:
  /Library/Frameworks/Python.framework/Versions/3.9/bin/python3.9-config
  /Library/Frameworks/Python.framework/Versions/3.9/bin/python3-config

**Reply by gromgit** [reply]
Given the results of your changes, you probably have a broken core tap. It should be an independent Git repo from the software repo (`/usr/local/Homebrew`), so try:
```
ls -l $(brew --repo homebrew/core)/.git/config
```
If you get a "File not found" or something similar, that pretty much confirms it, and you should then fix it with:
```
rm -fr $(brew --repo homebrew/core)
brew tap homebrew/core
```

**Reply by Bashert1601** [reply]
I get: 

-rw-r--r--  1 palashlalwani  admin  267 Mar  4 19:26 /usr/local/Homebrew/Library/Taps/homebrew/homebrew-core/.git/config

**Reply by gromgit** [reply]
Hmmm. What's the output of:
```
cat $(brew --repo homebrew/core)/.git/config
```
and
```
cat $(brew --repo)/.git/config
```
?

**Reply by Bashert1601** [reply]
Hey! The issue is solved after doing: 

rm -fr $(brew --repo homebrew/core)
brew tap homebrew/core 

Thanks for the help mate :)
